tests/tests_core/test_early_stopping.py

```diff
@@ -70,13 +72,27 @@ def test_bayes_factor(self):
 		stop,delta,CI,n_x,n_y,mu_x,mu_y = es.bayes_factor(self.rand_s1, self.rand_s2)
 		self.assertEqual(stop, 1)
 		self.assertAlmostEqual(delta, -0.15887364780635896)
-		self.assertAlmostEqual(CI['lower'], -0.24414725578976518)
-		self.assertAlmostEqual(CI['upper'], -0.072120687308212819)
+		self.assertAlmostEqual(CI['lower'], -0.25165623415486293)
```

Comment:
this assert seems to fail
E           AssertionError: -0.24873596733759926 != -0.25165623415486293 within 7 places